Sail/src/Sail/entities/components/MetaballComponent.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include "Component.h"
+#include "Sail/graphics/geometry/PhongMaterial.h"
+
+class MetaballComponent : public Component <MetaballComponent>{
+
+public:
+	MetaballComponent(float radius = 1.0f);
+	~MetaballComponent();
+	float getRadius() const;
+	void setRadius(float radius);
+	PhongMaterial* getMaterial();
```

Comment:
might need to be replaced with PBRMaterial (?)

Reply:
Not using the metaballs material yet, but i can change it.